.github/workflows/cli.yml

```diff
@@ -73,10 +78,13 @@ jobs:
         id: metadata
         uses: docker/metadata-action@v5
         with:
+          flavor: |
+            latest=false
```

Comment:
It only pushes on workflow dispatch so why can't we just turn this on and delete the conditional one below?

Reply:
Because the default one is triggered only when the workflow event is `tag`. We tried that before and it didn't work, which is why I added it here.

Reply:
Ah, I see. I didn't realize the defaults for the action don't apply it when it's a manual dispatch.